packages/lime-system/files/usr/lib/lua/lime/generic_config.lua

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/lua
+
+local config = require("lime.config")
+local utils = require("lime.utils")
+
+gen_cfg = {}
+
+gen_cfg.NODE_ASSET_DIR = '/etc/lime-assets/node/'
+gen_cfg.COMMUNITY_ASSET_DIR = '/etc/lime-assets/community/'
+gen_cfg.CONFIG_FIRST_BOOT_SIGNAL_FILE = '/etc/.cfg_first_boot_already_run'
+
+function gen_cfg.clean()
+    -- nothing to clean, but needs to be declared to comply with the API
+end
+
+function gen_cfg.configure()
+    gen_cfg.do_generic_uci_configs()
+    gen_cfg.do_copy_assets()
+    if not utils.file_exists(gen_cfg.CONFIG_FIRST_BOOT_SIGNAL_FILE) then
+        gen_cfg.do_run_assets('FIRSTBOOT')
```

Comment:
Should we use some enum like variable for `FIRSTBOOT` and `RECONFIG` ?

Reply:
done